app/serializers/search_result_serializer.rb

```diff
@@ -1,39 +1,12 @@
 class SearchResultSerializer < ActiveModel::Serializer
-  embed :ids, include: true
```

Comment:
This serialization is now a million times nicer.

Here's a sample serialized search result, which I think contains everything we'd need to put a nice UI for it together (and we can include anything else on the objects we'd like)

  {:priority=>"24.2",
   :query=>"loomio",
   :discussion=>
    {:id=>"2644",
     :blurb=>"<b>Loomio</b> Community! \r\n\r\nThis is a place where everyone can introduce themselves to the group, so we can all get to know",
     :title=>"Introduce yourself to the Loomio Community",
     :created_at=>Sat, 16 Mar 2013 00:28:57 UTC +00:00},
   :proposals=>[],
   :comments=>
    [{:id=>"206075", :blurb=>"<b>Loomio</b> right now to participate in the build up process of a new political party", :author=>"Hannah Salmon"},
     {:id=>"202156", :blurb=>"about it. I've been using <b>loomio</b> for a while and I find it very useful.", :author=>"Nico Aumar"}]}}

(Note that the <b> tags around 'loomio' are denoting that that's the term we searched for)